Library/Compiler.php

```diff
@@ -1260,7 +1260,12 @@ public function test(CommandInterface $command)
      */
     public function clean(CommandInterface $command)
     {
-        system('cd ext && make clean 1> /dev/null');
+        $this->fileSystem->clean();
+        if (Utils::isWindows()) {
+            system('cd ext && nmake clean-all');
+        } else {
+            system('cd ext && make clean > /dev/null');
+        }
```

Comment:
@sjinks Could we add something like this: https://github.com/phalcon/php-zephir-parser/blob/development/unit-tests/Extension/TestCase.php#L54-L77

related to the https://github.com/phalcon/zephir/issues/1462